src/main/webapp/app/shared/subject/subject-pair-dialog.component.ts

```diff
@@ -28,12 +29,14 @@ export class SubjectPairDialogComponent implements OnInit, OnDestroy {
     pairInfo: PairInfo = null;
     selectedClient: OAuthClient = null;
     allowPersistentToken = false;
+    public siteSettings$: Observable<SiteSettings>;
```

Comment:
Same as the above comment:
You can merge this line and line 51 here.
```
public siteSettings$ = this.siteSettingsService.getSiteSettings();
```
And you can remove the Observable import line on top and `public` keyword.